src/services/Functional/User.js

```diff
@@ -1,51 +1,41 @@
 import UserAPIService from "@/services/API/User.js";
-import UtilitiesService from "@/services/Functional/Utilities.js";
 
 export default {
-  reAuthenticate(store) {
-    // return the stored promise from the re-authentication call if re-authentication is in process
-    if (store.state.auth.isReAuthenticating) {
-      // to handle the case when isReAuthenticating has been set to true
-      // but no promise exists in the store. Log out the user in that case
-      if (
-        store.state.auth.reAuthenticationPromise == null ||
-        UtilitiesService.isObjectEmpty(store.state.auth.reAuthenticationPromise)
```

Comment:
do we need this function within UtilitiesService anymore?

Reply:
Not in this file, but this paritcular function is very helpful. It's just two lines and I wouldn't wanna search for this logic again from StackOverflow and write it. So i would suggest let it be

Reply:
We can always find it from our Github history if we need to, no?